spec/views/products/show.html.erb_spec.rb

```diff
@@ -1,5 +1,5 @@
 require "rails_helper"
 
-RSpec.describe "products/show.html.erb", type: :view do
+RSpec.describe "products/new.html.erb", type: :view do
```

Comment:
```suggestion
RSpec.describe "products/show.html.erb", type: :view do
```
I'm guessing this is a result of a RubyMine refactor. It should still be `"products/show.html.erb"` since that's the view the file name indicates is being tested.

Reply:
Yep, either that or I copy pasted it and forgot to change it haha, good catch!